src/layer/wms.ts

```diff
@@ -157,13 +157,17 @@ function getQueryParams(
     queryParams.format = params.format as MimeType;
   }
 
-  if (!params.fromTime) {
-    queryParams.time = moment.utc(params.toTime).format('YYYY-MM-DD');
-  } else {
+  // time is an optional param, and sentinelhub will atleast return the latest image if time=none https://docs.sentinel-hub.com/api/latest/api/ogc/wms/
+  // set time range with fromTime and toTime
+  if (params.fromTime !== null && params.toTime !== null) {
     queryParams.time = `${moment.utc(params.fromTime).format('YYYY-MM-DDTHH:mm:ss') + 'Z'}/${moment
       .utc(params.toTime)
       .format('YYYY-MM-DDTHH:mm:ss') + 'Z'}`;
   }
+  // Only toTime available. Requesting a single value for TIME parameter is deprecated
```

Comment:
Not sure if this comment should be kept or not.
If we decide to keep it, I would change the comment something similar to this 

```
// Other WMS providers may support a single value for TIME parameter, but Sentinel hub WMS service doesn't, so this should be handled in the application.
// it, so this needs to be handled
```

Reply:
Changed to suggested comment